lib/portus/ldap.rb

```diff
@@ -35,7 +35,7 @@ def authenticate!
         # strategy.
         if @ldap.bind_as(bind_options)
           user = find_or_create_user!
-          user.valid? ? success!(user) : fail!(:invalid_username)
+          user.valid? ? success!(user) : fail!(user.errors.full_messages.join(','))
```

Comment:
I'f I'm not wrong, you can just do this without calling `join`. This is already being handled by alerts.